yang/ietf-l3vpn-ntw.yang

```diff
@@ -2314,6 +2314,14 @@ module ietf-l3vpn-ntw {
                         description
                           "Indicates the IP address of the peer.";
                       }
+                      leaf virtual-ip-address {
```

Comment:
To be defined as a leaf-list

Note that this is defined as list virtual-ipv4-address and list virtual-ipv6-address in rfc8347, but I think that a leaf-list is more approrpiate.